views/movieDetails.ejs

```diff
@@ -0,0 +1,17 @@
+<div class="wrapper-results">
+	<div class="movie-details">
+	<!-- add ahref back button -->
+	<form action="/favorites" method="POST">
+		<button type="submit">Add to Favorites</button>
+			<input name="imdbID" type="hidden" value="<%= movie.imdbID %>" />
```

Comment:
This should be ```<input name="imdbId" type="hidden" value="<%= movie.imdbID %>" />``` where ```name``` is set to the column name of ```imdbId``` in your favorites table. If not, it won't write properly to the database.